workflows/quickstart/index.js

```diff
@@ -22,76 +22,71 @@ const workflowName = process.argv[4] || 'myFirstWorkflow';
 const searchTerm = process.argv[5] || null;
 
 // [START workflows_api_quickstart]
+
+// [START workflows_api_quickstart_client_libraries]
 const {ExecutionsClient} = require('@google-cloud/workflows');
 const client = new ExecutionsClient();
+// [END workflows_api_quickstart_client_libraries]
 
+// [START workflows_api_quickstart_execution]
 /**
- * TODO(developer): Uncomment these variables before running the sample.
- */
-// const projectId = 'my-project';
-// const location = 'us-central1';
-// const workflow = 'myFirstWorkflow';
-// const searchTerm = null;
-
-/**
- * Executes a Workflow and waits for the results with exponential backoff.
- * @param {string} projectId The Google Cloud Project containing the workflow
- * @param {string} location The workflow location
- * @param {string} workflow The workflow name
- * @param {string} searchTerm Optional search term to pass as runtime argument to Workflow
+ * Executes a Workflow
+ * @param {string} projectId The Google Cloud Project containing the Workflow
+ * @param {string} location The Workflow location
+ * @param {string} workflow The Workflow name
+ * @param {string} runTimeArgs Runtime arguments to pass to the Workflow as JSON string
  */
-async function executeWorkflow(projectId, location, workflow) {
-  /**
-   * Sleeps the process N number of milliseconds.
-   * @param {Number} ms The number of milliseconds to sleep.
-   */
-  function sleep(ms) {
-    return new Promise(resolve => {
-      setTimeout(resolve, ms);
-    });
-  }
-
-  // Execute workflow
+async function executeWorkflow(projectId, location, workflow, runtimeArgs) {
   try {
-    const runtimeArgs = searchTerm ? {searchTerm: searchTerm} : {};
     const createExecutionRes = await client.createExecution({
       parent: client.workflowPath(projectId, location, workflow),
       execution: {
-        // Provide runtime arguments as a JSON string
-        argument: JSON.stringify(runtimeArgs),
+        argument: runtimeArgs,
```

Comment:
If removing line 56 `const runtimeArgs = searchTerm ? {searchTerm: searchTerm} : {};` -  Would recommend having a default/ fallback similar to this, so users can safely not use that option. 
Also, this seems to need to still be in the format of a stringified json object `{searchTerm: searchTerm}` or for your changes `{searchTerm: runtimeArgs}` now.

Reply:
I moved the line out of the function to enclose it in its own region tag. See line 81. User should be able to run sample with or without a search term and it executes successfully